CONTRIBUTING.md

```diff
@@ -47,3 +47,73 @@ mvn -f sdk/{projectForlderDir}/pom.xml -Dgpg.skip clean install
 
 //example: mvn -f sdk/keyvault/azure-keyvault-keys/pom.xml clean install
 ```
+
+## Versions and versioning
+Tooling has been introduced to centralize versioning and help ease the pain of updating artifact versions in POM and README files. Under the eng\versioning directory there version text files, one for client (version_client.txt) and one for data (version_data.txt)*. The format of the version files is as follows:
```

Comment:
Consider linking to https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#consistent-dependency-versions

Reply:
Instead of linking to the one you'd provided, I'd added a link to the guidlines for incrementing after a release https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release and linked that under the "What does the process look like" heading.